src/helm/benchmark/static/schema.yaml

```diff
@@ -1910,19 +1923,113 @@ run_groups:
       - max_tokens
 
 ## Aspirational scenarios
+# Task coverage
+  - name: data_to_text_generation
+    display_name: Data-to-text generation 
+    description: Currently, we prioritize user-facing tasks in our core scenarios, but don't implement data-to-text generation. Could be implemented via WebNLG, E2E, ToTTo, etc.
+    taxonomy:
+      task: data-to-text generation
+    todo: true
+
+  - name: fact_verification
+    display_name: Fact verification 
+    description: Currently, we prioritize user-facing tasks in our core scenarios, but don't implement fact verification. Could be implemented via FEVER.
+    taxonomy:
+      task: fact verification
+    todo: true
+
+  - name: copy_writing
+    display_name: Copy writing 
```

Comment:
Should be one word "copywriting"

Reply:
Fixed, though fwiw I always felt like "copywriting" makes it look more similar to copyright + obfuscates that it is the practice of writing copy (even if that's the correct word). But changing the dictionary is a challenge for another day.